instructions.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
What's up with these DOCTYPE change? These don't seem desirable.

Reply:
This comes from the prettier update to v3 (see https://prettier.io/blog/2023/07/05/3.0.0.html#html). There's also no option to disable this behavior.

The doctype is case insensitive so IMO it's not important. I'm glad somebody else has an opinion about it so that I don't need to have one.